build/Dockerfile

```diff
@@ -79,25 +81,27 @@ ADD --link --chown=101:0 https://raw.githubusercontent.com/nginxinc/k8s-common/m
 ADD --link --chown=101:0 https://raw.githubusercontent.com/nginxinc/k8s-common/main/files/nap-dos-debian-11.repo nap-dos-11.sources
 ADD --link --chown=101:0 https://raw.githubusercontent.com/nginxinc/k8s-common/main/files/agent-debian-11.repo debian-agent-11.sources
 
-RUN --mount=from=busybox:musl,src=/bin/,dst=/bin/ printf "%s\n" "Acquire::https::pkgs.nginx.com::User-Agent k8s-ic-$IC_VERSION${BUILD_OS##debian-plus}-apt;" >> 90pkgs-nginx \
-	&& printf "%s\n" "user_agent=k8s-ic-$IC_VERSION${BUILD_OS##ubi*plus}-dnf" | tee -a nginx-plus-*.repo \
-	&& sed -i -e "s;%VERSION%;${NGINX_PLUS_VERSION};g" *.sources \
-	&& sed -i -e "y/0/1/" -e "1,8s;/centos;/${NGINX_PLUS_VERSION}/centos;" *.repo \
+RUN --mount=from=busybox:musl,src=/bin/,dst=/bin/ printf "%s\n" "Acquire::https::${PACKAGE_REPO}::User-Agent k8s-ic-$IC_VERSION${BUILD_OS##debian-plus}-apt;" >> 90pkgs-nginx \
+    && printf "%s\n" "user_agent=k8s-ic-$IC_VERSION${BUILD_OS##ubi*plus}-dnf" | tee -a nginx-plus-*.repo \
+	&& sed -i -e "s;%VERSION%;${NGINX_PLUS_VERSION};g" -e "s;pkgs.nginx.com;${PACKAGE_REPO};g" -e "s;${PACKAGE_REPO}/app-protect-security-updates;pkgs.nginx.com/app-protect-security-updates;g" *.sources \
+	&& sed -i -e "s;pkgs.nginx.com;${PACKAGE_REPO};g" 90pkgs-nginx \
+	&& sed -i -e "y/0/1/" -e "1,8s;/centos;/${NGINX_PLUS_VERSION}/centos;" -e "s;pkgs.nginx.com;${PACKAGE_REPO};g" -e "s;${PACKAGE_REPO}/app-protect-security-updates;pkgs.nginx.com/app-protect-security-updates;g" *.repo \
```

Comment:
```suggestion
RUN --mount=from=busybox:musl,src=/bin/,dst=/bin/ printf "%s\n" "Acquire::https::${PACKAGE_REPO}::User-Agent k8s-ic-$IC_VERSION${BUILD_OS##debian-plus}-apt;" >> 90pkgs-nginx \
    && printf "%s\n" "user_agent=k8s-ic-$IC_VERSION${BUILD_OS##ubi*plus}-dnf" | tee -a nginx-plus-*.repo \
	&& sed -i -e "s;%VERSION%;${NGINX_PLUS_VERSION};g" -e "s;pkgs.nginx.com;${PACKAGE_REPO};g" -e "s;pkgs.nginx.com/app-protect-security-updates;${PACKAGE_REPO}/app-protect-security-updates;g" *.sources \
	&& sed -i -e "s;pkgs.nginx.com;${PACKAGE_REPO};g" 90pkgs-nginx \
	&& sed -i -e "y/0/1/" -e "1,8s;/centos;/${NGINX_PLUS_VERSION}/centos;" -e "s;pkgs.nginx.com;${PACKAGE_REPO};g" -e "s;pkgs.nginx.com/app-protect-security-updates;${PACKAGE_REPO}/app-protect-security-updates;g" *.repo \
```

I had to reverse the order of `${PACKAGE_REPO}` and `pkgs.nginx.com` in the sed command for the app-protect-security-updates to get this to work. For some reason it would only update `pkgs.nginx.com` in the first occurrence of in the sources and repo files

For context
1. Before change:
```
Types: deb
URIs: https://pkgs-test.nginx.com/app-protect/R32/debian
Suites: bullseye
Components: nginx-plus
Signed-By: /usr/share/keyrings/nginx-archive-keyring.gpg
# security updates
Types: deb
URIs: https://pkgs.nginx.com/app-protect-security-updates/debian
Suites: bullseye
Components: nginx-plus
Signed-By: /usr/share/keyrings/app-protect-archive-keyring.gpg
```

After change:
```
# NAP WAF repos for debian 11 (bullseye).
# The VERSION variable needs to be replaced with the version of NGINX Plus you want to install, i.e. R30, or removed.
Types: deb
URIs: https://pkgs-test.nginx.com/app-protect/R32/debian
Suites: bullseye
Components: nginx-plus
Signed-By: /usr/share/keyrings/nginx-archive-keyring.gpg
# security updates
Types: deb
URIs: https://pkgs-test.nginx.com/app-protect-security-updates/debian
Suites: bullseye
Components: nginx-plus
Signed-By: /usr/share/keyrings/app-protect-archive-keyring.gpg
```